azure_functions_worker/testutils.py

```diff
@@ -410,10 +409,10 @@ async def communicate(self, message, *, wait_for):
         self._in_queue.put_nowait((message, wait_for))
         return await self._out_aqueue.get()
 
-    async def _start(self):
+    async def start(self):
```

Comment:
Are we using this **_start(self)** or **_close(self)** in other classes? I think keeping it as a private method should also work.

Reply:
`_MockWebHostController` uses these calls (which are defined in `_MockWebHost`)